plugins/sub_plugins/cli_parser/content_templates_parser.py

```diff
@@ -0,0 +1,83 @@
+"""
+content templates parser
+
+This is the content templates parser for use with the cli_parse module and action plugin.
+The parser functionality used by the network resource modules is leveraged here.
+
+"""
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+    author: Rohit Thakur (@rohitthakur2590)
+    name: content_templates
+    short_description: Define configurable options for C(content) sub-plugin of C(cli_parse) module
+    description:
+    - This plugin documentation provides the configurable options that can be passed
+      to the I(ansible.utils.cli_parse) plugins when I(ansible.netcommon.content_templates) is used as a value for
+      I(name) option.
+    version_added: 1.0.0
+"""
+
+EXAMPLES = r"""
```

Comment:
```suggestion
EXAMPLES = """
```

This is probably super nitpicky but I'm not sure why this is a raw string